l10n_it_corrispettivi_fatturapa_out/readme/DESCRIPTION.rst

```diff
@@ -1,7 +1,7 @@
 **Italiano**
 
-Questo modulo integra le funzionalità delle ricevute con la generazione della fatturazione elettronica.
+Questo modulo integra le funzionalità delle ricevute con la generazione della fatturazione elettronica. a sparire il bottone per la generazione dell'XML.
```

Comment:
Suggerisco

> Questo modulo integra le funzionalità delle ricevute con la generazione della fatturazione elettronica, nascondendo, per le ricevute, il bottone per la generazione dell'XML.

In caso, modifica anche l'inglese.
Infine rigenera il README.rst se puoi.

Grazie